CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+## [1.0.0] - 2023-05-04
```

Comment:
We may want to verify the versioning here first as the beta package may not be able to conform to semantic versioning.